flottform/server/src/database.ts

```diff
@@ -130,8 +177,11 @@ class FlottformDatabase {
 	}
 }
 
-export async function createFlottformDatabase(): Promise<FlottformDatabase> {
-	return new FlottformDatabase();
+export async function createFlottformDatabase(
+	cleanupPeriod = 30 * 60 * 1000,
+	entryTTL = 25 * 60 * 1000
```

Comment:
The default values are here a second time. Can we move them into a constant at the beginning of the file? `const DEFAULT_CLEANUP_PERIOD = 30 * 60 * 1000;` and `const DEFAULT_ENTRY_TIMEOUT_IN_MS = 25 * 60 * 1000`, then use
```suggestion
	cleanupPeriod = DEFAULT_CLEANUP_PERIOD,
	entryTTL = DEFAULT_ENTRY_TIMEOUT_IN_MS
```